sdk/dotnet/cmd/pulumi-language-dotnet/main.go

```diff
@@ -805,3 +814,96 @@ func (host *dotnetLanguageHost) GetProgramDependencies(
 		Dependencies: packages,
 	}, nil
 }
+
+func (host *dotnetLanguageHost) GenerateProject(
+	ctx context.Context, req *pulumirpc.GenerateProjectRequest) (*pulumirpc.GenerateProjectResponse, error) {
+	// Make a connection to the loader
+	conn, err := grpc.Dial(
+		req.LoaderAddress,
+		grpc.WithInsecure(),
+		rpcutil.GrpcChannelOptions(),
+	)
+	if err != nil {
+		return nil, errors.Wrapf(err, "language host could not make connection to loader")
+	}
+	loaderClient := codegenrpc.NewLoaderClient(conn)
+	loader := schema.NewGrpcLoader(loaderClient)
+	parser := hclsyntax.NewParser()
+	for file, source := range req.Source {
+		if err := parser.ParseFile(strings.NewReader(source), file); err != nil {
+			return nil, err
+		}
+		if parser.Diagnostics.HasErrors() {
+			return nil, parser.Diagnostics
+		}
+	}
+	bindOpts := []pcl.BindOption{
```

Comment:
I guess these should be sent over from import/convert? But they're a function of the ejector, not the pcl program itself it seems?